docs/guides/http2.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Leveraging HTTP/2
+description: Enhance web performance and secure communication by leveraging HTTP/2 with Tailcall.
+---
+
+HTTP/2 is a modern protocol that enhances improves web performance by introducing features like multiplexing, header compression, and more efficient connection handling.
+
+Tailcall seamlessly supports HTTP/2 on both the server (**ingress**) and client (**egress**) sides. This means that Tailcall facilitates the utilization of HTTP/2 protocol for both incoming requests to your server and outgoing requests from your server to other services.
```

Comment:
Perhaps a more concise version could be — 
Tailcall supports HTTP/2 for both server (ingress) and client (egress) operations, enabling the protocol for incoming and outgoing server requests.